api/onix.py

```diff
@@ -200,6 +200,10 @@ def parse(cls, file, data_source_name):
 
             subject_tags = parser._xpath(record, 'descriptivedetail/subject')
             subjects = []
+
+            # TODO: How much do we trust this subject data? We could
+            # bump up the weight quite a bit if it comes right from
+            # the publisher.
```

Comment:
Yeah, I think this is safe to bump up.

Reply:
Unlike the other changes, we can't migrate this change in an existing database because the subject types aren't distinctive and we don't know the data source.

Reply:
As far as I know I'm the only person who has used this so far, and only for one data source, so I can manually update mine.